modules/gltf/src/glb/glb-parser.js

```diff
@@ -81,8 +81,8 @@ export default class GLBParser {
 
   // Unpacks a bufferview into a new Uint8Array that is a view into the binary chunk
   getBufferView(glTFBufferView) {
-    const byteOffset = glTFBufferView.byteOffset + this.binaryByteOffset;
-    return new Uint8Array(byteOffset, glTFBufferView.byteLength);
+    const byteOffset = (glTFBufferView.byteOffset || 0) + this.binaryByteOffset;
```

Comment:
We might want to update `_resolveBufferView` to add this field if not present?

Reply:
Later, don't want to have both this and accessors at the same time.